Source/API/Interfaces.cs

```diff
@@ -522,4 +524,36 @@ public interface IAPI
 
         void RegisterPauseLock(PauseLockDelegate pauseLock);
     }
+
+    /// <summary>
+    /// Delegate for checking for any active <see cref="ThingFilterContext"/>
+    /// </summary>
+    /// <returns>Returns any active <see cref="ThingFilterContext"/></returns>
+    public delegate ThingFilterContext GetThingFilter();
+
+    public interface IThingFilterAPI
```

Comment:
Is IThingFilter name in use? Doesn't need API there.

Reply:
Well, I've went with the same approach as `IAPI`, and put API in the name - especially since it has similar purpose to `IAPI`, at least in my eyes. Though I can just change it if you're not a fan of putting API there.